docs/manual/java/gettingstarted/WhatsNext.md

```diff
@@ -0,0 +1,14 @@
+# What's Next?
+
+After working through "Getting started with Lagom", follow the path that make the most sense to you:
+
+* Learn how to integrate with Eclipse:
+    * [[Import an existing Maven project into Eclipse|EclipseMavenInt]] 
+    * [[Create a new Eclipse project using the Lagom Maven archetype|EclipseMavenNew]]
+    * [[Import an existing sbt build into Eclipse|EclipseSbt]]
```

Comment:
Tim, could you please clarify your question? Creating an new Eclipse project using the Maven archetype is easier than creating on the command line and importing (that's why I created a new topic for it). I don't think there is a way to do this for sbt, if that is what you are asking. And we do document how to import an sbt project into Eclipse.